### PR TITLE
first pass at v2 product spec

### LINKED ISSUE #48: feed this issues list with mods for v2

### PR DESCRIPTION
**Pull request by slively**
I took some liberties here so please treat it more as a proposal ready for review and editing.

closes #48

### CODE REVIEW

**Comment by jasepellerin** [comment]
So in the approval process, my understanding is that MVP doesn't want to be manually approving all of the projects. Would it be better to approve by default and send out the magic link in case the project needs to be deleted / changed? This way the site would operate autonomously and any input would be for curation rather than gatekeeping.

**Comment by mckelveygreg** [comment]
I love the direction this is going!! Go open source community projects!! 🚀

**Comment by slively** [comment]
> Would it be better to approve by default and send out the magic link in case the project needs to be deleted / changed? This way the site would operate autonomously and any input would be for curation rather than gatekeeping.

Not a bad idea, I'm willing to give that a shot if others feel the same way.

**Comment by slively** [comment]
I guess that also means there's another state of `rejected` for projects we want to remove.

**Comment by russell-osu** [comment]
If a project is rejected, how (and will) the author be notified? I assume they'll receive some sort of boilerplate notice, but will the reviewer be able to add comments or tick off a list of issues that motivated the rejection?

File: V2-PRODUCT-SPEC.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+#### Volunteers
+
+These are tech product and engineering folks with a range of experience from student to long time industry professional.
+They have extra time and a desire to learn while doing a public good.
+Industry pros have an opportunity to mentor, industry beginners get experience and hopefully a project on their resume, and 
+students get real experience for future internships and jobs.
+
+#### MVP Studio
+
+MVP Studio is primarily concerned with promoting a positive meeting space and being available for a project is ready to kick off.
```
Comment (oliverdain) [inline V2-PRODUCT-SPEC.md]: > being available for a project is ready to kick off.

Is grammer bad

File: V2-PRODUCT-SPEC.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    - stakeholder name
+    - stakeholder email (with a note that it will not be publicly visible)
+    - short project description (length of a tweet?)
+    - link to a longer project description (recommend they make a google doc or something?)
+        - it would be best if we didn't have a giant text field, or try to make a markdown thing
+- An email is sent to the stakeholder email
+    - The email contains a 'magic link' to a page where they can edit their posting
+    - The posting has an expiration of 2 months after it was posted.
+        - If it should be extended we can allow them to just re-post it
+- Once approved (see MVP Studio workflow below), the posting will be publicly visible
```
Comment (jasepellerin) [inline V2-PRODUCT-SPEC.md]: Posting visible immediately

File: V2-PRODUCT-SPEC.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+- On form submit 
+    - a thank you page is displayed with details on next steps again
+    - the stakeholder gets an email with the details of the volunteer
+
+### MVP Studio
+
+**The primary workflow for MVP Studio is to change the status of a project**
+
+- A project can be in one of the following statuses
+    - pending
```
Comment (jasepellerin) [inline V2-PRODUCT-SPEC.md]: Don't need pending, problematic posts will be deleted.

File: V2-PRODUCT-SPEC.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    - approved
+    - started
+    - finished
+- When a new project is submitted a 'magic link' is sent to team@mvpstudio.org
+    - the password in the magic link is different than the one sent to the stakeholder
+- Link goes to a page showing all the details of the new project.
+- There is a drop down to change the status.
+- when the status goes from pending to approved a congratulatory email is sent to the stakeholder.
```
Comment (jasepellerin) [inline V2-PRODUCT-SPEC.md]: Can send this email immediately

File: V2-PRODUCT-SPEC.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+- A project can be in one of the following statuses
+    - pending
+    - approved
+    - started
+    - finished
+- When a new project is submitted a 'magic link' is sent to team@mvpstudio.org
+    - the password in the magic link is different than the one sent to the stakeholder
+- Link goes to a page showing all the details of the new project.
+- There is a drop down to change the status.
```
Comment (jasepellerin) [inline V2-PRODUCT-SPEC.md]: In addition, there could be a button to remove the project

Comment (jasepellerin) [reply V2-PRODUCT-SPEC.md]: In addition, there needs to be a button to remove the project. Could also have a field to add reasons if we want to notify the NP that their project was rejected.

Comment (jasepellerin) [inline V2-PRODUCT-SPEC.md]: In addition, need a button to remove project. Could include a field to add reasons for removal if we want to email the NP.

File: V2-PRODUCT-SPEC.md (hunk 1 above)
Comment (slively) [reply V2-PRODUCT-SPEC.md]: is fixed

**Comment by slively** [comment]
> If a project is rejected, how (and will) the author be notified? I assume they'll receive some sort of boilerplate notice, but will the reviewer be able to add comments or tick off a list of issues that motivated the rejection?

Added a note about how if we want to reach out we will do it manually.

> I think the only thing that's missing is the bit where we get contacted by the project owner when they think they're ready to go and we help them get started.

Added notes that this should be part of the copy we provide before they submit a project on how they would kick off, and how they can contact us at any point.

File: V2-PRODUCT-SPEC.md (hunk 5 above)
Comment (slively) [reply V2-PRODUCT-SPEC.md]: per conversation since we will do it manually no need for this yet

File: V2-PRODUCT-SPEC.md (hunk 4 above)
Comment (slively) [reply V2-PRODUCT-SPEC.md]: updated

File: V2-PRODUCT-SPEC.md (hunk 3 above)
Comment (slively) [reply V2-PRODUCT-SPEC.md]: updated

File: V2-PRODUCT-SPEC.md (hunk 2 above)
Comment (slively) [reply V2-PRODUCT-SPEC.md]: updated

**Pull request merged**